src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -76,7 +76,10 @@ public virtual ICommandExecutive CommandExecutive
         {
             get { return commandExecutive ?? (commandExecutive = new ExtensionCommandExecutive(dynamoModel)); }
         }
-        
+
+        internal HostAnalyticsInfo HostInfo => dynamoModel.HostAnalyticsInfo;
```

Comment:
Would you add some comments to this and what it means?

Reply:
yes, I will, but note that I have made it internal on purpose for the time being.